pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -160,19 +155,10 @@ private ResourceUsage getMemUsage() {
 
     private boolean isPhysicalNic(Path path) {
         try {
-            path = Files.isSymbolicLink(path) ? Files.readSymbolicLink(path) : path;
-            if (!path.toString().contains("/virtual/")) {
-                try {
-                    Files.readAllBytes(path.resolve("speed"));
-                    return true;
-                } catch (Exception e) {
-                    // wireless nics don't report speed, ignore them.
-                    return false;
-                }
-            }
-            return false;
-        } catch (IOException e) {
-            LOG.error("Failed to read link target for NIC " + path, e);
+            Files.readAllBytes(path.resolve("speed"));
+            return true;
+        } catch (Exception e) {
+            // wireless nics don't report speed, ignore them.
```

Comment:
If you're 100% sure that it works, go ahead 😄 . Otherwise just pull back the if-not-virtual check.